src/core/client/admin/components/PaginatedSelect/PaginatedSelect.tsx

```diff
@@ -10,18 +16,21 @@ import {
   ButtonSvgIcon,
 } from "coral-ui/components/icons";
 import {
-  Button,
   ClickOutside,
   Dropdown,
   Flex,
   Popover,
   Spinner,
+  // TextField,
 } from "coral-ui/components/v2";
 
+import { useToggleState } from "coral-framework/hooks";
```

Comment:
Can you please move these imports up to the correct grouping?

Reply:
✅